MonacoEditorComponent/ts-helpermethods/Monaco.Helpers.KeyboardListener.js.map

```diff
@@ -0,0 +1 @@
+{"version":3,"file":"Monaco.Helpers.KeyboardListener.js","sourceRoot":"","sources":["Monaco.Helpers.KeyboardListener.ts"],"names":[],"mappings":"AAIA,GAAG"}
```

Comment:
We don't need the generated .js or .js.map files checked-in, eh?  As VS should output them when the build is run.

Reply:
Yes, that’s a good point. Will fix that when I come home later today.

Reply:
Fixed in https://github.com/hawkerm/monaco-editor-uwp/pull/26/commits/157cc9d7bd8902189829739111607ef81c7eca10